test/vfd_swmr_zoo_writer.c

```diff
@@ -346,44 +381,108 @@ main(int argc, char **argv)
             break;
         }
 
-        dbgf(1, "To reproduce, set seed (%s) to %u.\n", seedvar, seed);
+        dbgf(2, "To reproduce, set seed (%s) to %u.\n", seedvar, seed);
+
+        /* Writer tells reader to start */
+        notify = 1;
+        if (HDwrite(fd_writer_to_reader, &notify, sizeof(int)) < 0)
+            err(EXIT_FAILURE, "write failed");
 
         ostate = initstate(seed, vector, _arraycount(vector));
 
+        if (clock_gettime(CLOCK_MONOTONIC, &lastmsgtime) < 0)
+            errx(EXIT_FAILURE, "%s: clock_gettime", __func__);
+
```

Comment:
What are these`lastmsgtime` re-initializations for?

Reply:
We talked about it in our meeting.